modules/sublimeBidAdapter.js

```diff
@@ -125,7 +125,10 @@ export const spec = {
       data: {
         prebid: 1,
         request_id: requestId,
-        z: SUBLIME_ZONE
+        z: SUBLIME_ZONE,
+        meta: {
+          mediaTypes: bid.mediaTypes
```

Comment:
Is mediaTypes always exists ?
Maybe bid.mediaTypes || {} is safer ?

Reply:
We may change `{ meta: { mediaType: { banner: sizes... } } }` to something like `{ w: width, h: height }`.
It's easier to read than a stringified json and is cheaper in data transfer.